AS_923.yml

```diff
@@ -0,0 +1,38 @@
+band-id: AS_923
+uplink-channels:
+- frequency: 923200000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 923400000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+downlink-channels:
+- frequency: 923200000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 923400000
+  min-data-rate: 0
+  max-data-rate: 5
+lora-standard-channel:
+  frequency: 922100000
+  data-rate: 6
+  radio: 1
+fsk-channel:
+  frequency: 921800000
+  data-rate: 7
+  radio: 1
+radios:
+- enable: true
+  chip-type: SX1257
+  frequency: 923000000
+  rssi-offset: -166
+  tx:
+    min-frequency: 915000000
+    max-frequency: 928000000
+- enable: true
+  chip-type: SX1257
+  frequency: 922000000
+  rssi-offset: -166
```

Comment:
I think you can disable the second radio, since it isn't being used after deleting the high-bandwidth and FSK channels.